doc/modules/classes.rst

```diff
@@ -161,28 +196,45 @@ Classes
    :toctree: generated/
    :template: class.rst
 
-   cross_validation.KFold
-   cross_validation.LabelKFold
-   cross_validation.LabelShuffleSplit
-   cross_validation.LeaveOneLabelOut
-   cross_validation.LeaveOneOut
-   cross_validation.LeavePLabelOut
-   cross_validation.LeavePOut
-   cross_validation.PredefinedSplit
-   cross_validation.ShuffleSplit
-   cross_validation.StratifiedKFold
-   cross_validation.StratifiedShuffleSplit
+   model_selection.GridSearchCV
+   model_selection.RandomizedSearchCV
 
+Parameter search utilites
+-------------------------
+
+.. currentmodule:: sklearn
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   model_selection.ParameterGrid
+   model_selection.ParameterSampler
+
+Scorers
```

Comment:
I would rename this Model validation as scorers is more related to the metrics stuff.